integrations/visual_studio/cmake.rst

```diff
@@ -0,0 +1,196 @@
+.. _visual_cmake:
+
+|visual_logo| CMake Integration
+===============================
+
+Visual Studio 2017 comes with a CMake integration that allows one to just open a folder that contains a *CMakeLists.txt*
+and Visual will use it to define the project build.
+
+Conan can also be used in this setup to install dependencies. Let`s say that we are going to build an application that depends
+on an existing Conan package called ``hello/0.1@user/testing``. For the purpose of this example, you can quickly create this package by typing
+in your terminal:
+
+.. code-block:: bash
+
+    $ conan new hello/0.1 -s
+    $ conan create . user/testing # Default conan profile is Release
+    $ conan create . user/testing -s build_type=Debug
+
+The project we want to develop will be a simple application with these 3 files in the same folder:
+
+.. code-block:: cpp
+   :caption: **example.cpp**
+
+    #include <iostream>
+    #include "hello.h"
+
+    int main() {
+        hello();
+        std::cin.ignore();
+    }
+
+.. code-block:: text
+    :caption: **conanfile.txt**
+
+    [requires]
+    hello/0.1@user/testing
+
+    [generators]
+    cmake
+
+.. code-block:: cmake
+    :caption: **CMakeLists.txt**
+
+    project(Example CXX)
+    cmake_minimum_required(VERSION 2.8.12)
+
+    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
Legacy cmake integration

Reply:
I removed this whole section, because it's not relevant anymore